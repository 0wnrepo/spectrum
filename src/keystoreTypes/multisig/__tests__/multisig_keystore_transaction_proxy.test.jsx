import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import MultisigKeystoreTransactionProxy from '../multisig_keystore_transaction_proxy';

const contract = {
  address: '0x12345',
  submitTransaction: {
    getData: () => ({}),
  },
};

const addresses = [{
  address: '0x54325',
}, {
  address: '0x6423466',
}];

describe('<MultisigKeystoreTransactionProxy />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <MultisigKeystoreTransactionProxy
        contract={contract}
        addresses={addresses}
        txData={{}}
        web3={{}}
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
