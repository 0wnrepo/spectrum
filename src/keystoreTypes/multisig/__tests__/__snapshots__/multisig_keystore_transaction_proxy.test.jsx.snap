// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MultisigKeystoreTransactionProxy /> renders correctly 1`] = `
<p>
  Loading...
</p>
`;
